packages/blade/src/components/Typography/BaseText/types.ts

```diff
@@ -56,6 +56,7 @@ export type BaseTextProps = {
   fontStyle?: 'italic' | 'normal';
   textDecorationLine?: 'line-through' | 'none' | 'underline';
   lineHeight?: keyof Theme['typography']['lineHeights'];
+  wordBreak?: 'normal' | 'break-all' | 'keep-all' | 'break-word' | undefined;
```

Comment:
You can remove the `undefined` we are already setting this type optional `?:`

Reply:
```suggestion
  wordBreak?: 'normal' | 'break-all' | 'keep-all' | 'break-word';
```